ci/travis/36-minimal.yaml

```diff
@@ -14,6 +14,9 @@ dependencies:
   - pytest
   - pytest-cov
   - codecov
+  - fsspec
```

Comment:
This *does not* add `ffspec` as a dependency to the project, but does use file-like objects produced by it in the tests.